doc/changes/DM-41879.feature.rst

```diff
@@ -0,0 +1 @@
+Added functions to S3ResourcePath for generating presigned HTTP URLs
```

Comment:
I think you have added a new API to `ResourcePath` (say what it is) and there is only an implementation for S3.